components/_8x8_connect/_8x8_connect.app.mjs

```diff
@@ -8,4 +8,4 @@ export default {
       console.log(Object.keys(this.$auth));
     },
   },
-};
```

Comment:
Ignore all of these component changes. We should fix the underlying ESLint config so this doesn't happen, not urgent for this PR